test/core/rename/Rename_test.cpp

```diff
@@ -133,6 +133,14 @@ class olympia::LSUTester
         // while the ADD instruction is running, the STORE instruction should NOT issue
         EXPECT_TRUE(lsu.lsu_insts_issued_ == 0);
     }
+
+    void clear_entries(olympia::LSU &lsu){
```

Comment:
Granted this is a test, but the function name doesn't conform to standards

Reply:
@knute-sifive  I wanted to know if this approach of fixing the test is better or should I add a toggle to turn the notification off/on.
Both the approaches require the next person using the simulator to remember to drain  the lsu if they run short simulations for their tests like in this case.

Reply:
You could send the notification like Retire does if you know the test is good to end.

Reply:
Or call the callback explicitly in the tester

Reply:
I think the call is more explicit, but now everytime some one will write a functional or end to end test they would end up getting the error because they used the LSU in their test

Reply:
I think that's ok.  If someone wants to include the LSU in their functional test, they need to indicate to the LSU that the they believe the test is done.  Otherwise, the LSU might be indicating to them an error ... which is a good thing.